prov/util/src/util_mr_cache.c

```diff
@@ -399,6 +400,13 @@ struct ofi_mr_entry *ofi_mr_cache_find(struct ofi_mr_cache *cache,
 		goto unlock;
 	}
 
+	monitor = cache->monitors[entry->info.iface];
+	if (!monitor->valid(monitor, (const void *)entry->info.iov.iov_base,
+			    entry->info.iov.iov_len, &entry->hmem_info)) {
```

Comment:
Check the definition of `valid()`. The parameters here don't match the prototype.

Reply:
Thanks. Fixed.